lib/livebook/notebook.ex

```diff
@@ -69,6 +71,14 @@ defmodule Livebook.Notebook do
               url: String.t()
             }
 
+  @type output_panel_row :: %{items: list(output_panel_item)}
+
+  @type output_panel_item :: %{
+          id: Livebook.Utils.id(),
```

Comment:
This should be `cell_id`, otherwise it seems like just a unique item id :)

Reply:
I renamed it to id so I can use the helper function `access_by_id` in `Notebook.add_output_to_output_panel`

Reply:
We can make it `access_by_id(id, field \\ :id)` and pass `:cell_id`.